instruments/qubitekk/mc1.py

```diff
@@ -0,0 +1,185 @@
+        Enum for the motor types for the MC1
+        """
+        radio = "Radio"
+        relay = "Relay"
+
+    # PROPERTIES #
+
+    direction = int_property(
+        name="DIRE",
+        doc="""
+        Get the internal direction variable, which is a function of how far
+        the motor needs to go.
+
+        :type: `int`
+        """,
+        readonly=True
+    )
+
+    inertia = int_property(
+        name="INER",
+        doc="""
+        Gets/Sets the amount of force required to overcome static inertia.
+
+        :type: `int`
+        """,
+        set_fmt=":{} {}"
+    )
+
+    internal_position = int_property(
+        name="POSI",
+        doc="""
+        Get the internal motor state position.
+
+        :type: `int`
+        """,
+        readonly=True
+    )
+
+    metric_position = unitful_property(
+        name="METR",
+        doc="""
+        Get the estimated motor position, in microns.
+
+        :type: `quantities.`
+        """,
+        units=pq.mm,
+        readonly=True
+    )
+
+    setting = int_property(
+        name="OUTP",
+        doc="""
+        Gets/sets the output setting of the optical switch.
+
+        :type: `int`
+        """,
+        valid_set=range(2),
+        set_fmt=":{} {}"
+    )
+
+    step_size = int_property(
+        name="STEP",
+        doc="""
+        Gets/Sets the number of milliseconds per step.
+
+        :type: `int`
+        """,
+        valid_set=range(1, 100),
+        set_fmt=":{} {}"
+    )
+
+    @property
+    def firmware(self):
+        """
+        Gets the firmware version
+
+        :rtype: `tuple`(Major:`int`, Minor:`int`, Patch`int`)
+        """
+        # the firmware is assumed not to change while the device is active
+        # firmware is stored locally as it will be gotten often
+        # pylint: disable=no-member
+        if self._firmware is None:
+            while self._firmware is None:
+                self._firmware = self.query("FIRM?")
+                value = self._firmware.split(".")
+                if len(value) < 3:
+                    for _ in range(3-len(value)):
+                        value.append(0)
+                value = tuple(map(int, value))
+                self._firmware = value
+        return self._firmware
+
+    controller = enum_property(
+        'MOTO',
+        MotorType,
+        doc="""
+        Get the motor controller type.
+        """,
+        readonly=True
+    )
+
+    move_timeout = int_property(
+        name="TIME",
+        doc="""
+        Get the motor's timeout value, which indicates the number of clock
+        cycles before the motor can start moving again.
+
+        :type: `int`
+        """,
+        readonly=True
+    )
+
+    # METHODS #
+
+    def is_centering(self):
+        """
+        Query whether the motor is in its centering phase
+        :return: 0 if not centering, 1 if centering
+        :rtype: int
+        """
+        response = self.query("CENT?")
+        return int(response)
```

Comment:
Can I change this to

```python
return True if int(response) == 1 else False
```

It'll be more in line with all the other instruments then